cmake/AddRCInfo.cmake

```diff
@@ -30,6 +30,13 @@ if ("${git_branch}" STREQUAL "master")
 else()
   set(build_release ON)
 endif()
+set(original_filename "rime.dll")
+if(MSVC)
+  set(CMAKE_RC_FLAGS "${CMAKE_RC_FLAGS} -DMSVC")
+elseif(CMAKE_CXX_COMPILER_ID STREQUAL "GNU") # mingw
+  set(original_filename "librime.dll")
```

Comment:
general question: is keeping the "lib" prefix a convention in mingw?

Reply:
> general question: is keeping the "lib" prefix a convention in mingw?

I know nothing about this...